File: daytwo/javaScript/afterbreak/FathersAge.js
const ageCalculator = (fathersAge,sonsAge)=>{

let ageDifference = 0;

if(fathersAge > sonsAge * 2)
ageDifference = fathersAge - (sonsAge * 2);

else if (fathersAge < sonsAge * 2)
ageDifference = (sonsAge * 2) - fathersAge ;


return ageDifference ;

}


let daddyAge = 45;
let sonAge = 17;

wasTwiceAs = ageCalculator(daddyAge,sonAge);

console.log("the father was twice as old as his son: "+ wasTwiceAs + "years ago" );

let fathersAge = 30;
let boysAge = 17

willBeTwiceAs = ageCalculator(fathersAge,boysAge);

console.log("the father will be twice as old as his son in: "+ willBeTwiceAs + "years time" );

File: daytwo/python/afterbreak/Fathers_age.py
def age_calculator(fathers_age,sons_age):

	age_difference = 0
	
	if(fathers_age > sons_age * 2):
		age_difference = fathers_age - (sons_age * 2)	
	#return age_difference 
	
	elif(fathers_age < sons_age * 2):
		age_difference = (sons_age * 2)	- fathers_age
	
	
	return age_difference 	
	
	
daddy_age = int(34)	
	
ayo_age = int(15)


print("the father was  : ",age_calculator(daddy_age,ayo_age),"years ago")


#it works for both

	

